repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEntityStoreV2.java

```diff
@@ -1698,6 +1706,19 @@ private void executePreProcessor(EntityMutationContext context) throws AtlasBase
         }
     }
 
+    private void checkAndCreateProcessRelationshipsCleanupTaskNotification(AtlasEntityType entityType, AtlasVertex vertex) {
+        AtlasPerfMetrics.MetricRecorder metric = RequestContext.get().startMetricRecord("checkAndCreateAtlasDistributedTaskNotification");
+        try {
+            if (Arrays.asList(RELATIONSHIP_CLEANUP_SUPPORTED_TYPES).stream().anyMatch(type -> entityType.getTypeAndAllSuperTypes().contains(type))) {
```

Comment:
To avoid conversation per method call, we can initialize it as ArrayList in constructor

Reply:
Done